commands/auto/autonomousaction.py

```diff
@@ -23,17 +30,20 @@ class AutonomousRoutine(SequentialCommandGroup):
     def __init__(
         self,
         drive: DriveSubsystem,
+        arm: ArmSubsystem,
         name: str,
         simultaneousCommands: List[Command],
     ):
         self.name = name
         self.markerMap = {  # later todo: actual implementation
-            "store": WaitCommand(2),
-            "top": WaitCommand(2),
-            "mid": WaitCommand(2),
-            "hybrid": WaitCommand(2),
+            "store": SetArmPositionStored(arm),
+            "top": SequentialCommandGroup(SetArmPositionTop(arm), WaitCommand(0.8)),
```

Comment:
Why a wait on this one?

Reply:
give a bit of time once at the position for things to settle -- probably adding a wait to mid would be a good idea

Reply:
Doesn't it wait until it hits the position anyway? in the command or subsystem iirc it doesn't finish until the position is achieved

Reply:
it first hits the position based on motor stuff, I might need to adjust the position tolerences